### PR TITLE
React edit

### PR DESCRIPTION
**Pull request by stellardave**
- reviewed copy for all the docs in this section
- split automations page into two: _Use automations for dynamic responses_ and 						_Use custom triggers for automations_						
- edited automations pages

### CODE REVIEW

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 1]
```diff
@@ -1,22 +1,16 @@
...
-- Create a simple notification automation in just a few UI clicks
-- Build upon an event-based automation
-- Combine into a multi-layered responsive deployment pattern
-
-<Tip>
-**Available only on Prefect Cloud**
-
-Automations are a Prefect Cloud feature.
-</Tip>
+Potential actions include kicking on flow runs, pausing schedules, and sending custom notifications.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:13]:
```suggestion
Potential actions include starting flow runs, pausing schedules, and sending custom notifications.
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 2]
```diff
@@ -25,1249 +19,836 @@ Automations are a Prefect Cloud feature.
...
 
-3. Go to the automations page to create your first automation.
-![Automations page](/3.0rc/img/guides/automation-list.png)
+The [automations API](https://app.prefect.cloud/api/docs#tag/Automations) enables further programmatic customization of 
+trigger and action policies based on arbitrary [events](https://app.prefect.cloud/api/docs#tag/Events).
+</Note>
 
-4. Next, find the trigger type. In this case, use a flow completion. 
-![Trigger type](/3.0rc/img/guides/automation-triggers.png)
+Importantly, you can configure the triggers not only in reaction to events, but also proactively: to fire in the absence of
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:62]:
```suggestion
Importantly, you can configure the triggers not only in reaction to events, but also proactively: in the absence of 
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 3]
```diff
@@ -25,1249 +19,836 @@ Automations are a Prefect Cloud feature.
...
+example, the case of a cleanup flow you want to run or a specific notification you want to send.
 
-    To follow a more Python-based approach to create a deployment, use `flow.deploy` as in the example below.
+Inferred targets are deduced from the trigger itself.
 
-    ```python
-    # .deploy only needs a name, valid work pool 
-    # and a reference to where the flow code exists
+For example, if a trigger fires on a flow run that is stuck in a running state, and the action is to cancel an inferred 
+flow run—the flow run to cancel is inferred as the stuck run that caused the trigger to fire.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:139]:
```suggestion
flow run—the flow run that caused the trigger to fire.
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 4]
```diff
@@ -25,1249 +19,836 @@ Automations are a Prefect Cloud feature.
...
-  )
+Automation notifications support sending notifications through any predefined block that is capable of and configured 
+to send a message. That includes, for example:
 
-if __name__ == "__main__":
-    create_fake_artifact()  
-```
+- Slack message to a channel
+- Microsoft Teams message to a channel
+- Email to a configured email address
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:342]:
```suggestion
- Email to an email address
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 5]
```diff
@@ -25,1249 +19,836 @@ Automations are a Prefect Cloud feature.
...
+![Final notification](/3.0rc/img/guides/final-automation.png)
 
-Prefect defines three metrics today:
+<Tip>
+**No deployment created**
 
-- **Successes** (`{"name": "successes"}`), defined as the number of flow runs that went `Pending` and then the latest state we saw was not a failure (`Failed` or `Crashed`).  This metric accounts for retries if the ultimate state was successful.
-- **Duration** (`{"name": "duration"}`), defined as the _length of time_ that a flow remains in a `Running` state before transitioning to a terminal state such as `Completed`, `Failed`, or `Crashed`.  Because this time is derived in terms of flow run state change events, it may be greater than the runtime of your function.
-- **Lateness** (`{"name": "lateness"}`), defined as the _length of time_ that a `Scheduled` flow remains in a `Late` state before transitioning to a `Running` and/or `Crashed` state.  Only flow runs that the system marks `Late` are included.
+You do not need to create a deployment to trigger your automation, where a state outcome of a local
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:506]:
```suggestion
You do not need to create a deployment to trigger your automation. In the case above, the flow run state trigger fired in response to a flow run that was executed locally.
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 6]
```diff
@@ -25,1249 +19,836 @@ Automations are a Prefect Cloud feature.
...
 
-Prefect defines three metrics today:
+<Tip>
+**No deployment created**
 
-- **Successes** (`{"name": "successes"}`), defined as the number of flow runs that went `Pending` and then the latest state we saw was not a failure (`Failed` or `Crashed`).  This metric accounts for retries if the ultimate state was successful.
-- **Duration** (`{"name": "duration"}`), defined as the _length of time_ that a flow remains in a `Running` state before transitioning to a terminal state such as `Completed`, `Failed`, or `Crashed`.  Because this time is derived in terms of flow run state change events, it may be greater than the runtime of your function.
-- **Lateness** (`{"name": "lateness"}`), defined as the _length of time_ that a `Scheduled` flow remains in a `Late` state before transitioning to a `Running` and/or `Crashed` state.  Only flow runs that the system marks `Late` are included.
+You do not need to create a deployment to trigger your automation, where a state outcome of a local 
+flow run helped trigger this notification block. You are not required to create a deployment to trigger a notification.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:507]:
```suggestion
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 7]
```diff
@@ -25,1249 +19,836 @@ Automations are a Prefect Cloud feature.
...
-- [work_queue](/3.0rc/api-ref/server/schemas/core/#prefect.server.schemas.core.WorkQueue)
-- [work_pool](/3.0rc/api-ref/server/schemas/core/#prefect.server.schemas.core.WorkPool)
+You can create a deployment that triggers when a flow run takes longer than expected.
 
-In addition to its native properties, each object includes an `id` along with `created` and `updated` timestamps.
+### Use the Marvin library
 
-The `flow_run|ui_url` token returns the URL for viewing the flow run in Prefect Cloud.
+Take advantage of Prefect's [Marvin](https://www.askmarvin.ai/) library that uses an LLM to classify the data. 
+Marvin is great at embedding data science and data analysis applications within your pre-existing data engineering workflows.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:764]:
```suggestion
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 8]
```diff
@@ -25,1249 +19,836 @@ Automations are a Prefect Cloud feature.
...
-- [work_pool](/3.0rc/api-ref/server/schemas/core/#prefect.server.schemas.core.WorkPool)
+You can create a deployment that triggers when a flow run takes longer than expected.
 
-In addition to its native properties, each object includes an `id` along with `created` and `updated` timestamps.
+### Use the Marvin library
 
-The `flow_run|ui_url` token returns the URL for viewing the flow run in Prefect Cloud.
+Take advantage of Prefect's [Marvin](https://www.askmarvin.ai/) library that uses an LLM to classify the data. 
+Marvin is great at embedding data science and data analysis applications within your pre-existing data engineering workflows. 
+In this case, you can use [Marvin's AI functions](https://www.askmarvin.ai/docs/text/functions/) to help make the dataset more information rich.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:765]:
```suggestion
You can use [Marvin's AI functions](https://www.askmarvin.ai/docs/text/functions/) to make the dataset more information rich. 
```

File: docs/3.0rc/react/understand-events/custom-triggers.mdx [hunk 9]
```diff
@@ -0,0 +1,509 @@
...
+
+When you need a trigger beyond what the templates in the UI trigger builder provide, you can define a custom trigger in JSON. With custom triggers, you have access to the full capabilities of Prefect's automation system—allowing you to react to many kinds of events and metrics in your workspace.
+
+Each automation has a single trigger that, when fired, causes all of its associated actions to run. That single 
+trigger may be a reactive or proactive event trigger, a trigger monitoring the value of a metric, or a composite trigger 
+that combines several underlying triggers.
+
+### Event triggers
+
+Event triggers are the most common types of trigger. They are intended to react to the presence or absence of an event
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/custom-triggers.mdx:14]:
```suggestion
Event triggers are the most common type of trigger. They are intended to react to the presence or absence of an event.
```

File: docs/3.0rc/react/understand-events/custom-triggers.mdx [hunk 10]
```diff
@@ -0,0 +1,509 @@
...
+When you need a trigger beyond what the templates in the UI trigger builder provide, you can define a custom trigger in JSON. With custom triggers, you have access to the full capabilities of Prefect's automation system—allowing you to react to many kinds of events and metrics in your workspace.
+
+Each automation has a single trigger that, when fired, causes all of its associated actions to run. That single 
+trigger may be a reactive or proactive event trigger, a trigger monitoring the value of a metric, or a composite trigger 
+that combines several underlying triggers.
+
+### Event triggers
+
+Event triggers are the most common types of trigger. They are intended to react to the presence or absence of an event 
+happening in your workspace. Event triggers are indicated with `{"type": "event"}`.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/custom-triggers.mdx:15]:
```suggestion
Event triggers are indicated with `{"type": "event"}`.
```

File: docs/3.0rc/react/understand-events/custom-triggers.mdx [hunk 11]
```diff
@@ -0,0 +1,509 @@
...
+### Metric triggers
+
+Metric triggers (`{"type": "metric"}`) fire when the value of a metric in your workspace crosses a threshold you defined. 
+For example, you can trigger an automation when the success rate of flows in your workspace drops below 95% over the course 
+of an hour.
+
+Prefect's metrics are all derived by examining your workspace's events, and if applicable, use the `occurred` times of 
+those events as the basis for their calculations.
+
+Prefect defines three metrics today:
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/custom-triggers.mdx:255]:
```suggestion
Prefect defines three metrics:
```

File: docs/3.0rc/react/understand-events/custom-triggers.mdx [hunk 12]
```diff
@@ -0,0 +1,509 @@
...
+
+The `within` parameter for compound and sequence triggers restricts how close in time (in seconds) the child triggers 
+must fire to satisfy the composite trigger. For example, if the `daily-export-initiator` flow runs, but the other three 
+flows don't write their result files until three hours later, this trigger won't fire. Placing these time constraints 
+on the triggers can prevent a misfire if you know that the events will generally happen within a specific timeframe—
+and you don't want a stray older event included in the evaluation of the trigger. 
+
+If this isn't a concern for you, you may omit the `within` period, in which case there is no limit to how far apart in time the child triggers occur.
+
+You may compose any type of trigger into higher-order composite triggers, including proactive event triggers and metric
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/custom-triggers.mdx:447]:
```suggestion
You can compose any type of trigger into higher-order composite triggers, including proactive event triggers and metric 
```

**Review by billpalombi — COMMENTED** [review]
A few nits

**Comment by stellardave** [comment]
@billpalombi thanks for the feedback! Incorporated.

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 13]
```diff
@@ -1,22 +1,16 @@
...
-- Create a simple notification automation in just a few UI clicks
-- Build upon an event-based automation
-- Combine into a multi-layered responsive deployment pattern
-
-<Tip>
-**Available only on Prefect Cloud**
-
-Automations are a Prefect Cloud feature.
-</Tip>
+Potential actions include starting flow runs, pausing schedules, and sending custom notifications.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:13]:
```suggestion
Actions include starting flow runs, pausing schedules, and sending custom notifications.
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 14]
```diff
@@ -25,1249 +19,834 @@ Automations are a Prefect Cloud feature.
...
-from enum import Enum
-import pandas as pd
+Note that deployment triggers contribute to the total number of automations in your workspace.
 
-@ai_fn
-def generate_synthetic_user_data(build_of_names: list[dict]) -> list:
-    """
-    Generate additional data for userID (numerical values with 6 digits), location, and timestamp as separate columns and append the data onto 'build_of_names'. Make userID the first column
-    """
+## Automation notifications
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:333]:
```suggestion
## Sending notifications with automations
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 15]
```diff
@@ -25,1249 +19,834 @@ Automations are a Prefect Cloud feature.
...
-      key="fake-user-data",
-      table=artifact_df,
-      description= "Dataset that is comprised of a mix of autogenerated data based on user data"
-  )
+Notifications enable you to set up automation actions that send a message.
 
-if __name__ == "__main__":
-    create_fake_artifact()  
-```
+Automation notifications support sending notifications through any predefined block that is capable of and configured
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:337]:
```suggestion
Automations support sending notifications through any predefined block that is capable of and configured 
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 16]
```diff
@@ -25,1249 +19,834 @@ Automations are a Prefect Cloud feature.
...
-      table=artifact_df,
-      description= "Dataset that is comprised of a mix of autogenerated data based on user data"
-  )
+Notifications enable you to set up automation actions that send a message.
 
-if __name__ == "__main__":
-    create_fake_artifact()  
-```
+Automation notifications support sending notifications through any predefined block that is capable of and configured 
+to send a message. That includes, for example:
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:338]:
```suggestion
to send a message, including:
```

File: docs/3.0rc/react/understand-events/automations-triggers.mdx [hunk 17]
```diff
@@ -25,1249 +19,834 @@ Automations are a Prefect Cloud feature.
...
-def create_fake_user_dataset(df):
-  artifact_df = generate_synthetic_user_data(df)
-  print(artifact_df)
-  
-  create_table_artifact(
-      key="fake-user-data",
-      table=artifact_df,
-      description= "Dataset that is comprised of a mix of autogenerated data based on user data"
-  )
+Notifications enable you to set up automation actions that send a message.
```
Comment (billpalombi) [inline docs/3.0rc/react/understand-events/automations-triggers.mdx:335]:
```suggestion
```

**Review by billpalombi — COMMENTED** [review]
Simply notification section

**Review by billpalombi — APPROVED** [review]
lgtm

**Pull request merged**